### ISSUE TITLE
Abnormal amyloid beta protein level

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7068]
**Preferred term label:**

Abnormal amyloid beta protein level

**Synonyms**

Abnormal beta amyloid level, Abnormal Abeta level

**Definition (free text, please give PubMed ID)**
Any anomaly in the amount of the cleaved APP amyloid beta protein fragment.
PMID:31391004

**Parent term (use hpo.jax.org/app)**

Abnormality of the cerebrospinal fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added
